### ISSUE TITLE
Can gpl be used on Chinese models?

### ISSUE THREAD

**Issue by wduo** [UKPLab/gpl#2]
Great job!
Can I use gpl on Chinese models please? Which query generator model should I use? Which base models should be used? Which retrieval model should be used?
Looking forward to your reply. thanks.
@jcklie @reckart @dpetrak @nreimers @mbugert

**Comment by nreimers** [comment]
At the moment we have the doc2query model only for English. Also the Cross-Encoder is only available for English.

But they could be trained on this new dataset:
https://arxiv.org/abs/2203.10232

@kwang2049 What do you think, should we train doc2query & cross-encoder for Chinese?

**Comment by liushenglei** [comment]
I am very appreciate that if you could  train doc2query & cross-encoder for Chinese. Thx alot!

**Comment by nreimers** [comment]
@kwang2049 I think the DuReader dataset is good. But as I don't know Chinese, I'm not able to use that dataset as explanation etc. are mostly in Chinese.

**Comment by kwang2049** [comment]
> @kwang2049 @liushenglei @wduo Can you please keep me in the loop? I am also interested in the CN model. My email is [max@workhere.org](mailto:max@workhere.org). Can we connect? Thank you.

Yeah, sure. I think we can just write and post things here for now. I will also update it here if I got some new findings about this topic:)
